### PR TITLE
AAA 551 circuit breaker terraform

### PR DESCRIPTION
**Pull request by e-hat**

### CODE REVIEW

**Comment by e-hat** [comment]
I opted not to use a custom resource attribute check for the `duration` field as it was still possible to use the provided `TestCheckResourceAttr` to check that field.

File: provider/provider.go [hunk 1]
```diff
@@ -75,6 +76,34 @@ func OmitJsonArrayFields(val interface{}, omitList []interface{}) {
 	}
 }
 
+func timeSuffixToIntSec(tv string) int {
```
Comment (kaikaizi) [inline provider/provider.go:79]: maybe better named `timeWithUnitToSec`?

File: provider/provider.go [hunk 2]
```diff
@@ -75,6 +76,34 @@ func OmitJsonArrayFields(val interface{}, omitList []interface{}) {
...
+	switch tv[sz-1] {
+	case 's':
+		mult = 1
+	case 'm':
+		mult = 60
+	case 'h':
+		mult = 60 * 60
+	case 'd':
+		mult = 60 * 60 * 24
+	default:
```
Comment (kaikaizi) [inline provider/provider.go:95]: Maybe guard against unwanted non-numerical characters?

Comment (timrc-shore) [reply provider/provider.go:95]: Note: this is handled by `err` possibly returned from `strconv.Atoi()` below.

**Pull request merged**